src/api-v1.js

```diff
@@ -70,100 +70,106 @@ api.declare({
     // Intentionally vague because we don't want to reveafollowRedirects
     // our configuration too much
     let msg;
+    let code = 503; // default is that something is temporarily wrong
```

Comment:
not related to this PR... but I don't understand why you call `followRedirects` before even looking the cache...